src/plugins/vis_type_timeseries/public/application/components/vis_types/markdown/vis.js

```diff
@@ -42,13 +38,6 @@ function MarkdownVisualization(props) {
       }
     );
 
-    if (model.markdown_css) {
-      markdownCss = model.markdown_css.replace(
-        new RegExp(getMarkdownId(model.id), 'g'),
-        markdownElementId
-      );
-    }
-
     const markdownClasses = classNames('kbnMarkdown__body', {
```

Comment:
while replacing code with `@emotion`, try removing those awful classes. Maybe we can do it a little more professional

![image](https://user-images.githubusercontent.com/20072247/134036704-0e7dcde9-8b8b-424f-aaf8-0776ef51854a.png)

Reply:
Could you please provide ideas what we should do?